deepchem/feat/material_featurizers/composition_featurizer.py

```diff
@@ -0,0 +1,152 @@
+import re
+import numpy as np
+from collections import defaultdict
+
+from deepchem.utils.typing import PymatgenComposition
+from deepchem.feat import MaterialCompositionFeaturizer
+
+elements_tl = [
+    'H', 'Li', 'Be', 'B', 'C', 'N', 'O', 'F', 'Na', 'Mg', 'Al', 'Si', 'P', 'S',
+    'Cl', 'K', 'Ca', 'Sc', 'Ti', 'V', 'Cr', 'Mn', 'Fe', 'Co', 'Ni', 'Cu', 'Zn',
+    'Ga', 'Ge', 'As', 'Se', 'Br', 'Kr', 'Rb', 'Sr', 'Y', 'Zr', 'Nb', 'Mo', 'Tc',
+    'Ru', 'Rh', 'Pd', 'Ag', 'Cd', 'In', 'Sn', 'Sb', 'Te', 'I', 'Xe', 'Cs', 'Ba',
+    'La', 'Ce', 'Pr', 'Nd', 'Pm', 'Sm', 'Eu', 'Gd', 'Tb', 'Dy', 'Ho', 'Er',
+    'Tm', 'Yb', 'Lu', 'Hf', 'Ta', 'W', 'Re', 'Os', 'Ir', 'Pt', 'Au', 'Hg', 'Tl',
+    'Pb', 'Bi', 'Ac', 'Th', 'Pa', 'U', 'Np', 'Pu'
+]
+
+formulare = re.compile(r'([A-Z][a-z]*)(\d*\.*\d*)')
+
+
+class CompositionFeaturizer(MaterialCompositionFeaturizer):
+  """
+  Fixed size vector containing raw elemental compositions in the compound.
+
+  Returns a vector containing fractional compositions of each element
+  in the compound.
+
+  This featurizer requires the optional dependency pymatgen.
```

Comment:
Can you change like this?
https://github.com/deepchem/deepchem/blob/ab07c0b6b070d376039f55762e6894df289b7364/deepchem/feat/material_featurizers/element_property_fingerprint.py#L40-L42